skins/oasis/js/Tracking.js

```diff
@@ -60,16 +60,9 @@ jQuery(function ($) {
 		};
 	})();
 
-	/** article **/
+	/** page header (title-area) **/
 
 	(function () {
-		var category = 'article';
-
-		// Not special pages
-		if ($body.hasClass('ns-special')) {
-			return;
-		}
-
 		$('#WikiaPageHeader').on('mousedown', 'a', function (event) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has a complexity of 17 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)